src/uct/api/uct_def.h

```diff
@@ -111,6 +117,13 @@ typedef int                      uct_worker_cb_id_t;
     |<---- stride ----->|<---- stride ----->|
    @endverbatim
  *
+ * In order to describe an interleaved memory layout, each structure contains
```

Comment:
i've used UCP notation before, i'll try to be more clear how to do this in UCT:
ep_mem_reg { count=1, &memh_A}
ep_mem_reg { count=1, &memh_B}
ep_mem_reg { {memh_A, count=1}, {memh_B, count=1}, &memh_AB}
ep_mem_reg { {memh_AB, count=4},  &memh_ABABABAB}
ep_mem_reg { {memh_AB, count=2},  &memh_ABAB}